config/locales/en.yml

```diff
@@ -143,15 +143,21 @@ en:
       phone: Phone
     errors:
       duplicate_ssn: >
-        You may already have an account, please use that one.
-        You can always change your email address in your existing account.
+        An account already exists using the information you provided.
     messages:
       hardfail: >
         To access your information with the participating agency’s website,
         %{app_name} needs to verify your identity. We need to collect some basic
         personal information as well as some financial information from you to
         complete this process. If you don’t have the information we need at this
         time you can continue later.
+      dupe_ssn1: >
+        If you are getting this error, you may have already created and verified
+        and account, but with a different email address.
+      dupe_ssn2: Please %{link} with the email address you originally used.
+      dupe_ssn2_link: sign out now and sign back in
+      dupe_ssn3: "If you can't remember your original login information, you can %{link}."
```

Comment:
My vim editor syntax highlighting breaks when a string contains a single apostrophe, so I added the double quotes just to make my editor happy. YAML doesn't require double quotes afaik.